projects/plugins/jetpack/extensions/blocks/subscriptions/edit.js

```diff
@@ -92,6 +93,8 @@ export function SubscriptionEdit( props ) {
 		successMessage,
 	} = validatedAttributes;
 
+	const isButtonOnlyStyle = validatedAttributes.className === BUTTON_ONLY_CLASS_NAME;
```

Comment:
There's a helper function, example:

https://github.com/Automattic/jetpack/blob/67bcc94e6a60499f3ae900aff7cc6cd4123bb3e2/projects/plugins/jetpack/extensions/blocks/tiled-gallery/edit.js#L268

The className can be multiple classes, including custom ones inputed in editor by user.

Reply:
Fixed!